app/uk/gov/hmrc/cgtpropertydisposals/models/returns/SubmitReturnResponse.scala

```diff
@@ -31,6 +32,7 @@ object SubmitReturnResponse {
   final case class ReturnCharge(
     chargeReference: String,
     amount: AmountInPence,
+    processingDate: LocalDate,
```

Comment:
don't think you need this field here if it's in the `SubmitReturnResponse` already